testdata/stackTrace/app.js

```diff
@@ -1,9 +1,13 @@
 function buttonClick() {
-    setTimeout(inner, 100);
+    setTimeout(timeoutCallback, 100);
 }
 
-function inner() {
+function timeoutCallback() {
+    eval("evalCallback();");
+}
+
+function evalCallback() {
     (function() {
-        console.log('Inside anonymous function'); // bpLabel: stackTraceBreakpoint
```

Comment:
I still see that bpLabel referenced elsewhere, is that a problem?

Reply:
The framework tests (test/int/framework/*) still use the bpLabel approach and currently still need the labels. That is unrelated to these tests after this change.

Reply:
Oh ok